scripts/train.py

```diff
@@ -1,93 +1,145 @@
 # -*- coding: utf-8 -*-
 """
-Created on Thu May 14 11:38:50 2020
+Created on Thu June 14, 2020.
 
 @author: zhoul
 """
```

Comment:
IMO you don't really need a header like this. All this info is technically available in the git history